client/me/security-2fa-backup-codes/password-prompt.jsx

```diff
@@ -0,0 +1,68 @@
+import { localize } from 'i18n-calypso';
+import PropTypes from 'prop-types';
+import React from 'react';
+import FormButton from 'calypso/components/forms/form-button';
+import FormFieldset from 'calypso/components/forms/form-fieldset';
+import FormLabel from 'calypso/components/forms/form-label';
+import FormSettingExplanation from 'calypso/components/forms/form-setting-explanation';
+import FormTextInput from 'calypso/components/forms/form-text-input';
+
+class Security2faBackupCodesPasswordPromt extends React.Component {
+	static propTypes = {
+		onSubmit: PropTypes.func.isRequired,
+		onCancel: PropTypes.func.isRequired,
+	};
+
+	state = {
+		error: false,
+		userPassword: '',
+	};
+
+	componentDidMount = () => {
+		this.passwordInput.focus();
```

Comment:
Should we be using the `autoFocus` prop of `FormTextInput` instead? 

In any case, if we can avoid auto-focusing it would be best. Or if we really need to auto focus the field, let's do it and use `autoFocus` and intentionally disable the `jsx-a11y/no-autofocus` ESLint rule on that line. That can help if we ever decide to do a pass on improving the state of all those accessibility exceptions.

Reply:
Updated the code in https://github.com/Automattic/wp-calypso/pull/57560/commits/3dbb7178c35719161ae496a2d9d75a0603168d47